docs/Networking.asciidoc

```diff
@@ -85,9 +85,10 @@ run tests that require network connection between several machines (e.g.
 client -- server tests).
 
 The example of the configuration is applicable for openSUSE and will use _Open
-vSwitch_ for virtual switch, _SuSEfirewall2_ or _firewalld_ for NAT and
-_wicked_ as network manager. Keep in mind that a firewall is not strictly
-necessary for operation.
+vSwitch_ for virtual switch, _firewalld_ (or _SuSEfirewall2_ for older
+versions) for NAT and _wicked_ as network manager. Keep in mind that a
+firewall is not strictly necessary for operation but is not covered in all
```

Comment:
So you're saying operation *with* firewall or *without* firewall is not covered in all necessary details in this documentation? I would not connect these sentences with 'but' and instead just a new sentence.

Reply:
updated, please check again